File: backend/src/constants/env.py
import os
from src.utils.parse import parse

HTTP_WS_PORT = parse(os.environ.get('HTTP_PORT', ''), 8080, int)
ENFORCE_JWT_AUTH = parse(os.environ.get('ENFORCE_JWT_AUTH', ''), True, bool)
JWT_SECRET = os.environ.get('JWT_SECRET', '')

File: backend-python/src/constants/env.ts
import { tryParseBoolean, tryParseInt } from "utils/parse"

export const HTTP_WS_PORT = tryParseInt(process.env.HTTP_PORT ?? '', 10, 8080);

export const ENFORCE_JWT_AUTH = tryParseBoolean(process.env.ENFORCE_JWT_AUTH ?? true);
export const JWT_SECRET = process.env.JWT_SECRET ?? '';